src/lib/modules/query/hinter.js

```diff
@@ -17,31 +17,42 @@ class Hinter {
    * @param {Object} options
    * @param {Array<String>} options.collectionNames
    */
-  getHintsByValue(value, options) {
-    if (!value) return [];
+  getHintsByValue(fullQuery, currentWord, options) {
+    if (!fullQuery) return [];
     options = options && _.isObject(options) ? options : [];
 
     let hints = null;
+    let value = null;
 
-    value = value.trim ? value.trim() : value;
-    value = value.toLowerCase ? value.toLowerCase() : value;
+    value = fullQuery.trim ? fullQuery.trim() : fullQuery;
+    value = fullQuery.toLowerCase ? fullQuery.toLowerCase() : fullQuery;
 
-    let collectionName = parser.parseCollectionName(value);
-    // let functionName = parser.parseFunctionName(value);
+    let collectionName = parser.parseCollectionName(fullQuery);
+    let functionName = parser.parseFunctionName(fullQuery);
 
     var collectionNameRegex = collectionName ? new RegExp('[\\s\\S]+' + collectionName + '\.[\\s\\S]*') : null;
```

Comment:
I'm not able to reproduce this on my end, I'll put the new regexes in and see what i get.